### PR TITLE
#4144 issue: Update who-uses-dart page

### PR DESCRIPTION
**Pull request by MikiPAUL**
Hey @MaryaBelanger , I have made the changes as per the assigned issue. Please review the changes.

### CODE REVIEW

**Comment by MaryaBelanger** [comment]
Thanks @MikiPAUL! I'm going to pull this PR locally and work on it a little too, so let me know if you are planning to edit it further so I can wait and avoid merge conflicts. 

Otherwise, I'll push my additions in the upcoming week or so, and we can go over it together! Looking forward to iterating with you, and thanks again for authoring this contribution ❤️

**Comment by MikiPAUL** [comment]
Hey @MaryaBelanger, Thanks for taking the time to review my contribution and make changes to it. I appreciate your collaboration and look forward to reviewing the changes together. I don't have any edit plan and you can start work on it. Let me know if there's anything else I can do to help.

File: src/community/who-uses-dart.md [hunk 1]
```diff
@@ -6,21 +6,50 @@ toc: false
...
+[Google One]: https://one.google.com/about
+[Google Fibre]: https://fiber.google.com
+[Google Elections]: https://elections.google
+[Google Shopping]: https://shopping.google.com
+[Cloud Search]: https://workspace.google.com/intl/en_in/products/cloud-search
+
+Many companies use dart for building applications, such companies are:
+
+[Wrike][] uses the angulardart framework which is developed using dart, 
+[Blossom][], [ufirst][], [Modanisa][], [Sonos][], [Easy Insure][] and [Workviva][] uses for their web applications.
```
Comment (johnpryan) [inline src/community/who-uses-dart.md:47]:
```suggestion
[Blossom][], [ufirst][], [Modanisa][], [Sonos][], [Easy Insure][] and [Workiva][] uses for their web applications.
```

File: src/community/who-uses-dart.md [hunk 2]
```diff
@@ -6,21 +6,50 @@ toc: false
...
+
+[Wrike]: https://www.wrike.com
+[Sonos]: https://www.sonos.com/en-in/home
+[Blossom]: https://blossom.co
+[ufirst]: https://ufirst.com/en
+[Modanisa]: https://www.modanisa.com/en
+[Easy Insure]: https://easyinsure.ca/main.aspx
+[Workviva]: https://www.workiva.com/en-in
```
Comment (johnpryan) [inline src/community/who-uses-dart.md:55]:
```suggestion
[Workiva]: https://www.workiva.com/en-in
```

**Comment by MikiPAUL** [comment]
Hi @MaryaBelanger, I have made some updates to my pull request. I have fixed the typo. Please take a look at the pull request and let me know if you have any questions or concerns.

**Comment by MaryaBelanger** [comment]
@MikiPAUL I had to make some changes to adhere to our formatting and style guidelines, and I needed to create a new PR to do so. I need to close this one now, but I have added you as an author on the new PR, so you'll get authorship credit when it merges! Thanks for getting this started!

**Pull request closed without merging**